helper.py

```diff
@@ -69,6 +69,7 @@ def attach_sensor(robot, sensor_name, metalog):
                 function2 = SourceLogger(None, remission_log_name).get
         else:
             robot.laser = LaserIP(remission=True)
+            robot.laser.stopOnExit = False  # for faster boot-up
```

Comment:
I don't really understand the naming here - `stopOnExit`? If it is `False` something should not stop on exit. That something is laser. So, is stop actually a shutdown? And exit is end of the program? I don't have a much better name though :worried:

Reply:
Yes, it is on sys.exit() of the program - it is rather "legacy" variable (taken from Eduro).

Reply:
Are you planning to handle somehow the act of shutting down everything? I remember one of the problems was not waiting for the threads to really finish/exit.